types/index.test-d.ts

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
run into some issues with dtslint, then just converted it to tsd to align it with other repos.